src/scripts/main.js

```diff
@@ -1,8 +1,31 @@
 'use strict';
 
 const people = require('./lib/people');
+// write your code here
+const table = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.map(el => {
+  const tr = document.createElement('tr');
+  const nameTd = document.createElement('td');
+  const genderTd = document.createElement('td');
+  const bornTd = document.createElement('td');
+  const diedTd = document.createElement('td');
+  const ageTd = document.createElement('td');
+  const centuryTd = document.createElement('td');
 
-// write your code here
+  nameTd.textContent = el.name;
+  genderTd.textContent = el.sex === 'm' ? 'Male' : 'Female';
+  bornTd.textContent = el.born;
+  diedTd.textContent = el.died;
+  ageTd.textContent = el.died - el.born;
+  centuryTd.textContent = Math.ceil(el.died / 100);
+
+  tr.append(nameTd);
+  tr.append(genderTd);
+  tr.append(bornTd);
+  tr.append(diedTd);
+  tr.append(ageTd);
+  tr.append(centuryTd);
```

Comment:
```suggestion
  tr.append(nameTd, genderTd, bornTd);
```
Добавляй все одним append'ом